common/lib/capa/capa/templates/choicetext.html

```diff
@@ -62,15 +65,17 @@
 
     <div class="indicator-container">
     % if input_type == 'checkbox' or not element_checked:
-        <span class="status ${status.classname}" id="status_${id}"></span>
+        <span class="status ${status.classname}" id="status_${id}">
+            <span class="sr">${status.display_name}</span>
+        </span>
     % endif
     </div>
 
     % if show_correctness == "never" and (value or status not in ['unsubmitted']):
     <div class="capa_alert">${_(submitted_message)}</div>
     %endif
     % if msg:
-    <span class="message">${msg|n}</span>
+    <span class="message" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
@sstack22 and @cptvitamin FYI
No linking of the "message" is being done here, as there does not appear to be a good element to link to.  

This problem type is unsupported and problematic in other ways.
http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/mult_choice_num_input.html?highlight=choicetextresponse